File: CORREDOR/4 Lenguajes/JS/Parametros_Retorno.js
class Parametros_Retorno {
    // Método con retorno
    decirAlgo(nombre){
        return "Hola " + nombre;
    }
    multiplicar(x, y) {
        return x * y; // devuelve el resultado
    }

    Division(x, y) {
        return x / y;
    }

    suma(x, y) {
        return x + y;
    }

    resta(x, y) {
        return x - y;
    }
}

const calc = new Parametros_Retorno();

console.log(calc.decirAlgo ("Johan"));

const resultado = calc.multiplicar(4, 2);
console.log("La multiplicación es: " + resultado);

const resultado2 = calc.Division(40, 10);
console.log("La división es: " + resultado2);

const resultado3 = calc.suma(8, 15);
console.log("La suma es: " + resultado3);

const resultado4 = calc.resta(15, 20);
console.log("La resta es: " + resultado4);

File: CORREDOR/4 Lenguajes/Java/Parametros_Retorno.java
package Java;

public class Parametros_Retorno {
    // Método con retorno
        int multiplicar(int x, int y) {
        return x * y; // devuelve el resultado
    }
        int Division(int x, int y){
        return x / y;
    }
        int suma(int x, int y){
        return x + y;
    }
        int resta(int x, int y){
            return x - y;
        }

    public static void main(String[] args) {
        Parametros_Retorno calc = new Parametros_Retorno();

        int resultado = calc.multiplicar(4, 2);
        System.out.println("La multiplicación es: " + resultado); 

        int resultado2 = calc.Division(40, 10);
        System.out.println("la division es: " + resultado2);

        int resultado3 = calc.suma(8, 15);
        System.out.println("La suma es: " + resultado3);

        int resultado4 = calc.resta(15, 20);
        System.out.println("La resta es: " + resultado4);
    }
}